ros2bag/ros2bag/verb/play.py

```diff
@@ -152,6 +153,11 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
                  'By default, if loaned message can be used, messages are published as loaned '
                  'message. It can help to reduce the number of data copies, so there is a greater '
                  'benefit for sending big data.')
+        parser.add_argument(
+            '--service-request-from', default='service_introspection',
+            choices=['service_introspection', 'client_introspection'],
+            help='Determine the source of the service request to be replayed. '
+                 'By default, the service request is from recorded service introspection message.')
```

Comment:
@Barry-Xu-2018 I am considering to rename the `--service-request-from` parameter and `service_request_from` play option to the "--service-request**s**-**source**" and "play_options.service_request**s**_**source**" respectively.
Also use short names for enum values without `INTROSPECTION` postfix. 
i.e., `ServiceRequestsSource.SERVICE` and `ServiceRequestsSource.CLIENT`.
IMO '"from"'  postfix usually is not used for parameters or variables, the `source` is a better fit for that purpose.

If you agree I can make changes on my branch as a separate commit.

Reply:
@MichaelOrlov 

I agree with your suggestions. Thanks！